src/react-components/src/components/02-molecules/Cards/EditionCard.tsx

```diff
@@ -12,7 +12,8 @@ export type EditionDetails = {
   language: string,
   license: string,
   readOnlineLink: string,
-  downloadLink: string
+  downloadLink: string,
+  noLinkElement?: JSX.Element
```

Comment:
I'd recommend adding inline comments like 
```
/** Element to render when there are no links. */
noLinkElement?: JSX.Element
```
mostly because this prop is a bit hard to read.